src/openms/source/FORMAT/MzTab.cpp

```diff
@@ -3768,15 +3792,16 @@ Not sure how to handle these:
     if (psm_id_ >= peptide_ids_.size()) return false;
     const PeptideIdentification* pid = peptide_ids_[psm_id_];
     auto psm_row = MzTab::PSMSectionRowFromPeptideID_(
-      *pid, 
-      prot_ids_, 
-      idrunid_2_idrunindex_,
-      map_id_run_fileidx_2_msfileidx_,
-      run_to_search_engines_,
-      psm_id_, 
-      db_, 
-      db_version_,
-      export_empty_pep_ids_);
+        *pid,
+        prot_ids_,
+        idrunid_2_idrunindex_,
+        map_id_run_fileidx_2_msfileidx_,
+        run_to_search_engines_,
+        0u,
```

Comment:
is this correct?

Reply:
yes since not supported for consensus yet.